networkx/algorithms/tests/test_euler.py

```diff
@@ -112,3 +112,15 @@ def test_multigraph_with_keys(self):
     @raises(nx.NetworkXError)
     def test_not_eulerian(self):
         f = list(eulerian_circuit(nx.complete_graph(4)))
+
+class TestEulerize(TestCase):
+
```

Comment:
You should probably add the corner cases like ```G=nx.Graph()``` and ```G=nx.empty_graph(3)```.
Also, test that it raises an error for a directed graph.